src/Symfony/Component/Console/Tests/ApplicationTest.php

```diff
@@ -452,6 +452,36 @@ public function testFindAlternativeNamespace()
         }
     }
 
+    public function testFindAlternativesOutput()
+    {
+        $application = new Application();
+
+        $application->add(new \FooCommand());
+        $application->add(new \Foo1Command());
+        $application->add(new \Foo2Command());
+        $application->add(new \Foo3Command());
+
+        $expectedAlternatives = array(
+            'afoobar',
+            'afoobar1',
+            'afoobar2',
+            'foo1:bar',
```

Comment:
The good thing is that in real apps we probably won't have this problem. This example (`foo:`, `foo1:`, `foo3:`) is too abstract and probably our users don't use `project:, `project1:` or `app:`, `app1:` namespaces.